include/ola/Clock.h

```diff
@@ -40,8 +40,10 @@
 
 namespace ola {
 
+static const uint64_t NSEC_IN_SEC = 1000000000;
 static const int USEC_IN_SECONDS = 1000000;
-static const int ONE_THOUSAND = 1000;
+static const int MSEC_IN_SEC = 1000;
+static const int ONE_THOUSAND = MSEC_IN_SEC;
```

Comment:
Please add a deprecation notice with the date to ONE_THOUSAND and ensure it's not used anywhere else in the codebase any more (common/thread/Mutex.cpp).

Reply:
Actually looking at the code more, ONE_THOUSAND is still in regular use, so scrap that.

Reply:
renamed all